unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/SkyboxCamera.cs

```diff
@@ -9,6 +9,19 @@ public class SkyboxCamera
         private Camera skyboxCamera;
         private SkyboxCameraBehaviour camBehavior;
 
+        public SkyboxCamera(Camera mainCam)
```

Comment:
This whole class is doing stuff like setting the camera to a rendering asset without Post Processing, modifying `renderShadows`, changing the layer and a bunch of super dangerous stuff that's being apply to the main camera. The whole script  shouldnt be needed anymore.

Reply:
Thanks, I will go more aggressive on that 👍